collect_app/src/main/java/org/odk/collect/utilities/Result.java

```diff
@@ -5,7 +5,10 @@
 /**
  * Based on https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-result/ which is not
  * available to Java.
+ *
+ * @deprecated use {@link org.odk.collect.shared.result.Result} instead
  */
+@Deprecated
```

Comment:
Again, I would have done the work to replace this instead of deprecating here, but wanted to avoid the extra risk given that this will be going into a hotfix.